openqa-label-known-issues

```diff
@@ -17,14 +17,14 @@ label_on_issue() {
 }
 
 main() {
-    [ "$dry_run" = "1" ] && client_prefix=echo
+    [ "$dry_run" = "1" ] && client_prefix="echo"
     client_call="${client_call:-"$client_prefix openqa-client --host $host_url"}"
     for i in $(cat - | sed 's/ .*$//'); do
         id="${i##*/}"
         out=$(mktemp)
-        if ! curl -s "$i/file/autoinst-log.txt" -o "$out"; then
+        if [ $(curl -s -w "%{http_code}" "$i/file/autoinst-log.txt" -o "$out") != "200" ]; then
```

Comment:
This need quotes around the `$(...)` or just use double `[[` which I would prefer.

Reply:
all done